assets/js/prosemirror/schema.js

```diff
@@ -2,9 +2,8 @@ import { Schema } from 'prosemirror-model';
 import { generateSchemaMarks } from './marks';
 import blocks from './blocks';
 
-export default (options) => (
+export default (options) =>
   new Schema({
-    nodes: blocks(options.blocks),
-    marks: generateSchemaMarks(options.marks),
-  })
-);
+    nodes: blocks(options.blocks, options.inline),
```

Comment:
```suggestion
    nodes: blocks(options),
```
maybe we can do something like this.
and on the receiving end 

```javascript
blocks({blocks, inline})
```

Reply:
It's a javascript object... it would be `{block: '', inline: ''}`
But indeed, if you the receiving end is an elixir it would be something that we could pattern match anyway `%{block: block, inline: inline}`

Reply:
No it would just grab out the blocks and inline
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#syntax